test/e2e/specs/all-features/record/presentationRelatedTab.spec.js

```diff
@@ -0,0 +1,91 @@
+var chaisePage = require('../../../utils/chaise.page.js');
+
+var EC = protractor.ExpectedConditions;
+var testParams = {
```

Comment:
What is the purpose of defining this whole `testParams` object? It looks like you are only using a small subset of these values. There's no reason to bloat the spec with unnecessary variable definitions.